### ISSUE TITLE
PROPOSAL: New Core Team Role: Membership Lead

### ISSUE THREAD

**Issue by de-la-viz** [plocket/cfb-hacking-project-management#30]
# Why

On-boarding new members really drains the resources of projects and project managers. At the same time, many members that projects do get are not skilled enough to be engaged in the work and projects are missing people with specific skills that are necessary for moving forward. Finally, the ownership/responsibility of the recruiting process and onboarding experience is unclear. 

# How

Create a role in the core team (that can be open for now) for Membership Lead to manage finding new members.  

The core team ‘membership lead’ role identifies skill gaps and project needs across the CfB portfolio, and decides on the best strategy to tackle them. They take ownership of the recruiting and onboarding experience.  

Working with project leads and Core Team, they - for instance - develop an outreach, onboarding, and communications strategies that maximize CfB visibility within targeted groups, such as project management organizations and meetups or relevant listserves; while also creating processes that would encourage recruitment and retention of productive members.

# Resources

# Proposal History

## 2020/02/12
1. Proposal issue made with 'proposal' label and other relevant labels based on the discussion on February 7 and the first draft made in Google Drive.

**Comment by plocket** [comment]
I like a lot of this! Is the 'on-boarding' bit new? I don't remember hearing of it/seeing it before. And does it need to be in this role? If not, I might separate them out. Later, if one person wants to do both roles, they can choose to do that.

**Comment by de-la-viz** [comment]
Mmmh not sure myself... I tried to summarize our discussion based not only on the text in google drive, but also based on the discussion btw you @moss and @Morisy in the Slack channel. 
Personally, I would say that it is inherently linked because we onboard who we recruit. So good coordination, clarity on who does what is needed. 
There are probably two onboarding processes, one is the general CfB onboarding and the other occurs inside each team. This role could encompass the global CfB onboarding and ensure a smooth transition to the team. I think this reflects your idea of having “global CfB tutorials (Slack, git, ...)” for recruits. 
That said, it is true that we could also imagine separating them out.

**Comment by plocket** [comment]
Yeah, the two are definitely related. I do think if they were separate, it'd allow people to be more flexible. Are you comfortable enough with that idea to edit the proposal to show that change?
